tests/random_schema.cc

```diff
@@ -959,14 +959,14 @@ data_model::mutation_description::key random_schema::make_ckey(uint32_t n) {
     return make_clustering_key(n, g);
 }
 
+// Return up to `n` random clustering keys.
+// Note: may return fewer than n keys if the schema limits the clustering keys space.
```

Comment:
This should be in the declaration site (in the header).

Reply:
right, done